R/nullModelApproxSE.R

```diff
@@ -0,0 +1,217 @@
+## estimate the ratio (se.ratio) of the true SE(score) to the fast SE(score)
```

Comment:
@smgogarten @amstilp - I'm still stuck on good names. The to be exported functions are currently called:
- fitNullModelApproxSE
- calcScoreSE
- updateNullModApproxSE

and the input parameter for `assocTestSingle` is called `approx.score.SE` (T/F). 

Another thought I had was to use "fast" instead of "approx". So maybe something like `fitNullModelFastSE`, `updateNullModFastSE`, and `fast.score.SE`.

But I'm open to any name suggestions.